src/js/components/DataTable/Header.js

```diff
@@ -310,6 +323,18 @@ const Header = forwardRef(
                 );
               }
 
+              if (verticalAlign || columnVerticalAlign) {
+                const vertical = verticalAlign || columnVerticalAlign;
+                let justify = 'center';
```

Comment:
Can we reuse `verticalAlignToJustify` from TableCell by chance?

Reply:
Switched this to use `verticalAlignToJustify` in latest commit